packages/gatsby-tinacms-git/src/use-git-form.ts

```diff
@@ -46,8 +46,16 @@ export function useGitForm<N extends GitNode>(
       return gitFile.show()
     },
     onSubmit: gitFile.commit,
-    reset: gitFile.reset,
+    reset: () => {
+      if (options.reset) {
+        options.reset()
+      }
+      return gitFile.reset()
+    },
     onChange({ values }: any) {
+      if (options.onChange) {
+        options.onChange(values)
+      }
```

Comment:
While `<FormOptions />` allowed for `reset` and `onChange` to be passed in, they were ignored.  This alters the behavior to more closely match standard `useForm()`.